math/pythagoras.py

```diff
@@ -0,0 +1,16 @@
+"""
+input two of the three side in right angled triangle and return the third. use "?" to indicate the unknown side. 
+"""
+
+def pythagoras(opposite,adjacent,hypotenuse):
```

Comment:
should also handle the case when user enters more than one side as '?'. The result cannot be tabulated in this case.